ServiceNow/JFrog Spoke/README.md

```diff
@@ -14,7 +23,7 @@ If you need help with this integration, please contact `support@jfrog.com`
 
 ServiceNow Spokes are applications with predefined actions that customers can use to build workflows on the NOW Platform. The JFrog Spoke has actions that can be combined with other native Spokes from ServiceNow and other third-party tools to create enterprise-grade workflows for vulnerability and change management for all your ITSM needs.
 
-<img src="https://github.com/jfrog/partner-integrations/blob/main/ServiceNow/JFrog%20Spoke/images/im-1.png?raw=true" width="800">
+<img src="./images/HelloWorldFlow.png?raw=true" width="800">
```

Comment:
I think you don't need to use `img` tag for image. See https://gist.github.com/uupaa/f77d2bcf4dc7a294d109 for examples.

Reply:
Is it okay to leave the tag? I like it because it makes it clear when reading the code what it is. Like .1 sec recognition vs .5 sec, its nicer for my brain

Reply:
Sure, not a blocker.